pandas/tests/resample/test_deprecated.py

```diff
@@ -278,3 +278,36 @@ def test_resample_base_with_timedeltaindex():
 
     tm.assert_index_equal(without_base.index, exp_without_base)
     tm.assert_index_equal(with_base.index, exp_with_base)
+
+
+def test_interpolate_posargs_deprecation():
+    # GH 41485
+    df = DataFrame({"ds": ["1992-08-27 07:46:48", "1992-08-27 07:46:59"], "y": [1, 4]})
+    s = Series(
+        df.iloc[:, 1].values.reshape(-1), index=pd.to_datetime(df.iloc[:, 0].values)
+    )
+
+    msg = (
+        r"In a future version of pandas all arguments of DataFrame\.interpolate "
+        r"except for the argument 'method' will be keyword-only"
+    )
+
+    with tm.assert_produces_warning(FutureWarning, match=msg):
+        result = s.resample("3s").interpolate("linear")
```

Comment:
need to construct `expected` and check that `result` matches it, `tm.assert_series_equal`